test/inverse.jl

```diff
@@ -0,0 +1,11 @@
+using StatsFuns
```

Comment:
Can you add the file to the tests in `test/runtests.jl`?

Reply:
Ooops, indeed. Done.